<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QMainWindow" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前选择的排序模式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="mode_chosen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="num0">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num1">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num2">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num3">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num4">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num5">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num6">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num7">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num8">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num9">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num10">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num11">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="read_file">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件读入</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num12">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>100</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据输入方式：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="manual_input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>手动输入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_file">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>38</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_path">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="execute_sort">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>130</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pre_step">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>160</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上一步</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_step">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>160</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一步</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_result">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>130</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reChoose">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重新选择数据</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>耗时：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>步骤数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>当前步骤数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_spend">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="step_num">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="now_step">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>排序算法选择</string>
    </property>
    <addaction name="action_direct_insert"/>
    <addaction name="action_half_insert"/>
    <addaction name="action_shell"/>
    <addaction name="action_bubble"/>
    <addaction name="action_quick"/>
    <addaction name="action_easy_choose"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_direct_insert">
   <property name="text">
    <string>直接插入排序</string>
   </property>
  </action>
  <action name="action_half_insert">
   <property name="text">
    <string>折半插入排序</string>
   </property>
  </action>
  <action name="action_shell">
   <property name="text">
    <string>希尔排序</string>
   </property>
   <property name="iconText">
    <string>希尔排序</string>
   </property>
  </action>
  <action name="action_bubble">
   <property name="text">
    <string>冒泡排序</string>
   </property>
  </action>
  <action name="action_quick">
   <property name="text">
    <string>快速排序</string>
   </property>
  </action>
  <action name="action_easy_choose">
   <property name="text">
    <string>简单选择排序</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
